docker-centos/config.json.template

```diff
@@ -323,7 +333,7 @@
         },
 	{
 	    "type": "bind",
-	    "source": "${STATE_DIRECTORY}",
+	    "source": "/var/lib/docker-$NAME",
```

Comment:
what about using `/var/lib/$NAME`?  In this way if we install a container called `docker` we are able to use the existing storage (if any) used by Docker on the host